### PR TITLE
context_manager level one

### PR DESCRIPTION
**Pull request by kerbez**
am I doing right?

### CODE REVIEW

File: context_manager/instruction.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Context managers are a way of allocating and releasing some sort of resource exactly where you need it. The most widely used example of context managers is the 'with' statement.
```
Comment (An4ik) [inline context_manager/instruction.md]: let's instructions.md to be only instructions.
I mean here we should have instructions  what student has to do.
No need information about  Context manager here
only steps:
1. do this 
2. do this

File: context_manager/tests/context_manager_test.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from unittest import TestCase
 import os
-import cantext_manager as f
+import context_manager as f
```
Comment (An4ik) [inline context_manager/tests/context_manager_test.py]: let's import directly without as f

from context_manager import write_wihtou

Comment (kerbez) [reply context_manager/tests/context_manager_test.py]: tests for first level is tests/tests_for_level_one.py. context_manager_test.py is old vertion, I will delete it later

File: context_manager/write_into_file.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+def write_with_context_manager(filename):
+    """
+    Writes 'write with context manager done!' into filename.
+    :param filename: txt file
+    :return: boolean, true if file closed
+    """
+
+
+"""
+GeneratorContextManager object.
```
Comment (An4ik) [inline context_manager/write_into_file.py]: this is not a docstring (docs) remove it please

File: context_manager/instruction.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+```
+### LEVEL-1.
+
+1. Look at the file write_into_file.py
+
+2. Look at the tests in tests folter 
+
+3. Run tests with command:
+
+    ```python3 -m unittest tests/tests_for_level_one.py```
```
Comment (An4ik) [inline context_manager/instruction.md]: test's filename should be exactly the same with the file you are testing.

in your case:
  test_write_into_file.py

**Pull request closed without merging**